app/src/handlers/census/events/AchievementEarnedEvent.ts

```diff
@@ -0,0 +1,40 @@
+import { injectable } from 'inversify';
+import { GenericEvent } from '../../../types/censusEventTypes';
+import { AchievementEarned } from 'ps2census/dist/client/utils/PS2Events';
+import Parser from '../../../utils/parser';
+import IllegalArgumentException from '../../../exceptions/IllegalArgumentException';
+import ZoneUtils from '../../../utils/ZoneUtils';
+import { Zone } from '../../../constants/zone';
+
+@injectable()
+export default class AchievementEarnedEvent {
+    public readonly worldId: number;
+    public readonly characterId: number;
+    public readonly zone: Zone;
+    public readonly achievementId: number;
+    public readonly timestamp: number;
+
+    public constructor(
+        event: GenericEvent
```

Comment:
Just thought actually, we should probably change generic event to `CensusEvent`, better naming for it.

Reply:
PS2Census may want to change it then

Reply:
Oh yeah that's coming from there. I'll mention it to microwave.